charts/postgresql/values.yaml

```diff
@@ -0,0 +1,14 @@
+architecture: replication
+global:
```

Comment:
All of the values inside `values.yaml` currently have no effect. If you wish to override the defaults provided inside the upstream values.yaml, here: https://github.com/bitnami/charts/blob/main/bitnami/postgresql/values.yaml, then you need to indent them under a top-level `postgresql:` section.

Reply:
Fixed 👍